src/components/ApplicationDetails/tabs/overview/visualization/hooks/useAppWorkflowData.ts

```diff
@@ -135,7 +118,6 @@ export const useAppWorkflowData = (
       componentGroup,
       buildGroup,
       testsGroup,
-      staticEnvironmentGroup,
       ...(mvpFeature ? [] : [releaseGroup]),
       ...(mvpFeature ? [] : [managedEnvironmentGroup]),
```

Comment:
I think we need to remove this as well.

Reply:
managedEnvironment is ReleasePlan resource.